meteor-app/imports/startup/server/parts-data.js

```diff
@@ -0,0 +1,41 @@
+import { Meteor } from 'meteor/meteor' // base
+import Parts from '/imports/api/parts/parts'
+import casual from 'casual' 
+
+
+Meteor.methods({
+'seed.parts'() {
+  const n = 10
+// seed ensures same data is generated
+  casual.seed(123)
+
+  const arrayOf = function (times, generator) {
+    let result = [];
+    for (let i = 0; i < times; ++i) {
+      result.push(generator());
+    }
+    return result;
+  };
+
+  casual.define('part', function () {
+    return {
+      imageUrl: '/public/images/logo-large.jpg',
```

Comment:
This isn't going to be pretty, all the parts will have the same picture - can't you provide a set of image urls and randomly select from them?